tdrs-backend/zap-scanner.sh

```diff
@@ -1,28 +1,20 @@
 #!/bin/sh
 
-if command -v sha256sum >/dev/null ; then
-	SHASUM=sha256sum
-elif command -v shasum >/dev/null ; then
-	SHASUM=shasum
-fi
-
-docker-compose down
-docker-compose up -d --build
-    # do an OWASP ZAP scan
-     export ZAP_CONFIG=" \
-      -config globalexcludeurl.url_list.url\(0\).regex='.*/robots\.txt.*' \
-      -config globalexcludeurl.url_list.url\(0\).description='Exclude robots.txt' \
-      -config globalexcludeurl.url_list.url\(0\).enabled=true \
-      -config spider.postform=true"
+# do an OWASP ZAP scan
```

Comment:
Due to the increased timeout referenced above, the previous code that was bringing all the containers down and rebuilding them from scratch caused ZAP to run before web had completed the startup process. This down/build/up process has long been unnecessary, since we are already building the containers from scratch in an earlier step.